src/translate.py

```diff
@@ -3643,9 +3895,26 @@ def process_instr(instr: Instruction) -> None:
         if mnemonic in arch.instrs_ignore:
             pass
 
-        elif mnemonic in arch.instrs_store:
+        elif mnemonic in arch.instrs_store or mnemonic in arch.instrs_store_update:
             # Store a value in a permanent place.
-            to_store = arch.instrs_store[mnemonic](args)
+            if mnemonic in arch.instrs_store:
+                to_store = arch.instrs_store[mnemonic](args)
+            else:
+                # PPC specific store-and-update instructions
+                # `stwu r3, 8(r4)` is equivalent to `$r3 = *($r4 + 8); $r4 += 8;`
+                to_store = arch.instrs_store_update[mnemonic](args)
+
+                # Update the register in the second argument
+                update = args.memory_ref(1)
+                if not isinstance(update, AddressMode):
+                    raise DecompFailure(
+                        f"Unhandled store-and-update arg in {instr}: {update!r}"
+                    )
```

Comment:
This error occurs for all storex's, right? Would be nice to unify code for that with loadx. (We can fix this later.)